lib/coophub/backends/backends.ex

```diff
@@ -131,6 +131,21 @@ defmodule Coophub.Backends do
     end
   end
 
+  @spec get_rate_limit(<<_::48, _::_*16>>) :: any
+  def get_rate_limit(source) do
+    backend = get_backend_module!(source)
+    {_, url, headers} = backend.prepare_request_rate_limit()
```

Comment:
WARNING: Although for now this function it's only called from `Backends.get_rate_limit("github")` so it's only implemented for github, the call here `backend.prepare_request_rate_limit()` should be threated as any other behaviour functions to be consistent.
Suggestion: In my opinion, we should define the callback at `Backends.Behaviour` and implement it in the others backends also, but since ie gitlab doesn't has any limit we could return `:unlimited` or something like that and not make any request here (we could return `-1` from this `get_rate_limit/1` function) and let the caller know that there is no limit.
That will allow us to change/refactor/remove the specific case at `CacheWarmer.get_from_github/1`, `get_data("github", github_orgs)` and the `merge_github_orgs` to apply the `cached_at` logic to any org/backend and not just for github.

Reply:
That's a good call, but not sure if implementing generally, since it has a read from cache for each org in the `merge_github_orgs/1`.